src/asset_hub.rs

```diff
@@ -126,15 +100,12 @@ impl AssetHub {
         Ok(cursor)
     }
 
-    pub fn get_metadata<'a, K, V: DBTransaction<'a, T>, T: lmdb::Transaction + 'a>(
+    pub fn get_metadata<'a, V: DBTransaction<'a, T>, T: lmdb::Transaction + 'a>(
```

Comment:
I know you didn't add this but I think the compiler can infer this lifetime now in 2018?

Reply:
Maybe it could here, I haven't tried. But anyway, this should probably not be done as a part of this PR.

Reply:
👍 Yeah, just kinda stuck out to me I guess I'm too indoctrinated in 2018 edition.